### PR TITLE
Update hw edit, create, show views

### PR DESCRIPTION
**Pull request by bradweee**
### Changelog
- Updated HW views to match current course sessions views
- Added admin permissions to edit HW
- Create get_lecture_name method to display lecture name on hw page

### Checklist:
- [x]  My code follows the style guideline
- [x]  I have performed a self-review of my own code
- [ ]  I have made corresponding changes to the documentation
- [ ]  Add tests if required

### UI Screenshots:
**Homework Create View**
new_hw

**Homework Edit View**
edit_hw

**Homework Show View**
view_hw

### CODE REVIEW

File: app/views/homeworks/new.html.erb [hunk 1]
```diff
@@ -1,5 +1,7 @@
 <h1>New Homework</h1>
```
Comment (shaolinhoward) [inline app/views/homeworks/new.html.erb:1]: same here

File: app/views/homeworks/edit.html.erb [hunk 2]
```diff
@@ -1,6 +1,8 @@
 <h1>Editing Homework</h1>
```
Comment (shaolinhoward) [inline app/views/homeworks/edit.html.erb:1]: can we also update the header while at it? lets use H2's and center it. see the mentor view for example

File: app/views/homeworks/new.html.erb (hunk 1 above)
Comment (bradweee) [reply app/views/homeworks/new.html.erb]: Fixed!

File: app/views/homeworks/edit.html.erb (hunk 2 above)
Comment (bradweee) [reply app/views/homeworks/edit.html.erb]: Fixed!

**Pull request merged**